docs/src/getting-started/quick-start.md

```diff
@@ -0,0 +1,84 @@
+# Quick Start
+
+Get the Miden Faucet running in minutes.
+
+## Prerequisites
+
+- Miden Faucet installed (see [Installation](./installation.md))
+- Access to a Miden node (testnet or local)
+
+## Step 1: Create a Faucet Account
+
+```bash
+miden-faucet create-faucet-account \
+  --output-path ./faucet.mac \
+  --token-symbol MIDEN \
+  --decimals 6 \
+  --max-supply 100000000000000000
+```
```

Comment:
Most of this is self explanatory, but I would add a short paragraph before this to explain what is going to happen. That is, we are creating a faucet account with the specified token symbol, decimals etc.). The crated account data will be written to `faucet.mac`. This will not yet deploy the account to the network (it will be deployed at some point later on) etc.

This also applies to other steps in this file.